barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplemini.hpp

```diff
@@ -326,18 +423,13 @@ template <typename Curve> class ShpleminiVerifier_ {
 
         if (has_zk) {
             commitments.emplace_back(hiding_polynomial_commitment);
-            scalars.emplace_back(-unshifted_scalar); // corresponds to ρ⁰
+            scalars.emplace_back(-claim_batcher.get_unshifted_batch_scalar()); // corresponds to ρ⁰
```

Comment:
wonder whether negation here can be somehow encompassed in the functionality of claim_batcher

Reply:
I'll think about this when I think about the inclusion of rho / rho power in the batcher